### PR TITLE
read / write grid files

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

**Comment by woshilapin** [comment]
It might be nice to add them to an integration test like `ntfs2ntfs` just to see if we parse them correctly, and write them correctly.

File: src/objects.rs [hunk 1]
```diff
@@ -1440,6 +1440,62 @@ impl AddPrefix for TicketUseRestriction {
...
+        deserialize_with = "de_from_date_string",
+        serialize_with = "ser_from_naive_date"
+    )]
+    pub end_date: Date,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GridRelCalendarLine {
+    pub calendar_id: String,
+    #[serde(skip)]
```
Comment (woshilapin) [inline src/objects.rs]: Not sure I understand why this field is skipped?

Comment (patochectp) [reply src/objects.rs]: files in production don't have this field

**Comment by patochectp** [comment]
I am waiting for the validation of the specifications

**Comment by woshilapin** [comment]
I updated this PR, following the specifications. More to come on the topic of `grid_*.txt` files for:
- prefixes
- merge
- sanitize

**Pull request merged**